.eslintrc.js

```diff
@@ -1,20 +1,60 @@
-/**@type {import('eslint').Linter.Config} */
-// eslint-disable-next-line no-undef
-module.exports = {
-  // root: true,
-  // parser: '@typescript-eslint/parser',
-  // plugins: [
-  // 	'@typescript-eslint',
-  // ],
-  // extends: [
-  // 	'eslint:recommended',
-  // 	'plugin:@typescript-eslint/recommended',
-  // ],
-  // rules: {
-  // 	'semi': [2, "always"],
-  // 	'@typescript-eslint/no-unused-vars': 0,
-  // 	'@typescript-eslint/no-explicit-any': 0,
-  // 	'@typescript-eslint/explicit-module-boundary-types': 0,
-  // 	'@typescript-eslint/no-non-null-assertion': 0,
-  // }
+/** @type {import('eslint').Linter.Config} */
+const config = {
+  extends: [
+    'eslint:recommended',
+    'plugin:eslint-comments/recommended',
+    'plugin:@typescript-eslint/recommended',
+  ],
+  env: {
+    browser: true,
+    node: true,
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+    project: [
+      './tsconfig.eslint.json',
+      './apps/**/tsconfig.json',
+      './packages/*/tsconfig.json',
+    ],
+  },
```

Comment:
`@typescript-eslint` allows us to configure linting to be handled on a per-project basis. In this case, we set `projects` as a root `tsconfig.eslint.json` file (which will cover all non-package files like configs, scripts, and test files) and and the `tsconfig.json` files from each of our packages.